extensions/applicationinsights-debugplugin-js/package.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "@microsoft/applicationinsights-debugplugin-js",
+    "version": "2.5.4",
```

Comment:
nit: it should be 2.5.5, but I'll update it as part of the next release to keep in-sync with the primary release numbers